wire-ios/Wire-iOS Tests/ConversationList/Container/ConversationListViewControllerSnapshotTests.swift

```diff
@@ -84,6 +85,14 @@ final class ConversationListViewControllerTests: XCTestCase {
             selfProfileViewControllerBuilder: .mock
         )
         sut.mainSplitViewState = .collapsed
+        // sut.navigationController?.navigationBar.barTintColor = SemanticColors.View.backgroundDefaultWhite
+
+        // Set the navigation bar color here
+           let navBarAppearance = UINavigationBarAppearance()
+           navBarAppearance.backgroundColor = SemanticColors.View.backgroundDefaultWhite
+           UINavigationBar.appearance().standardAppearance = navBarAppearance
+           UINavigationBar.appearance().scrollEdgeAppearance = navBarAppearance
```

Comment:
Won't this set all nav bar appearances? Won't this affect other tests?

Reply:
Done: https://github.com/wireapp/wire-ios/commit/2c669b9a98137eb0ce0aa305e9e3d9731d5bed5d

Deleted all the setup styling code for now. Still trying to figure out why the generated ones come with a transparent nav bar background color.